tools/mo/openvino/tools/mo/moc_frontend/pipeline.py

```diff
@@ -35,19 +35,20 @@ def moc_pipeline(argv: argparse.Namespace, moc_front_end: FrontEnd):
                         "Please use use_legacy_frontend=True to convert the model.")
     else:
         input_checkpoint = getattr(argv, 'input_checkpoint', None)
+        enable_mmap = not argv.disable_mmap
```

Comment:
```suggestion
        enable_mmap = not getattr(argv, 'disable_mmap', False)
```

Reply:
This way you do not need to update MO unit-tests below

Reply:
good catch, thanks  ;)